src/header.ce.vue

```diff
@@ -192,11 +192,11 @@ onMounted(() => {
           <a
             class="link-btn"
             href="/console/account/userdetails"
-            :title="state.user?.username"
+            :title="state.user?.firstname + ' ' + state.user?.lastname"
```

Comment:
I prefer more modern string interpolation syntax for the concatenations here and below.
```suggestion
            :title="`${state.user?.firstname} ${state.user?.lastname}`"
```